packages/date-time-utilities/src/timeMath/timeMath.ts

```diff
@@ -0,0 +1,32 @@
+import { TimeConstants } from '../dateValues/timeConstants';
+
+/**
+ * Returns a date offset from the given date by the specified number of minutes.
+ * @param date - The origin date
+ * @param minutes - The number of minutes to offset. 'minutes' can be negative.
+ * @returns A new Date object offset from the origin date by the given number of minutes
+ */
+export const addMinutes = (date: Date, minutes: number): Date => {
+  const result = new Date(date.getTime());
+  result.setTime(result.getTime() + minutes * TimeConstants.MinutesInOneHour * TimeConstants.MillisecondsIn1Sec);
+  return result;
+};
+
+/**
+ * Returns the earliest available minute based on the increments
+ * @param minute - Minute to ceil
+ * @param increments - Time increments
+ * @returns Ceiled minute
+ */
+export const ceilMinute = (minute: number, increments: number) => {
```

Comment:
personal preference, but maybe `ceilMinuteToIncrement` would be more descriptive?

Reply:
Makes sense to me! updated